designs/jane/src/body.mjs

```diff
@@ -0,0 +1,219 @@
+
+    points.neckLeft = points.middle.shiftTowards(points.leftShoulder, neckWidth / 2)
+    points.neckRight = points.middle.shiftTowards(points.rightShoulder, neckWidth / 2)
+
+    points.middleHem = points.bottomLeft.shiftTowards(points.bottomRight, widthBody / 2)
+
+    points.neckFront = points.middle
+      .shiftTowards(points.middleHem, measurements.hpsToBust * options.neckDepthFront)
+      .attr('data-text', 'Front Neckline')
+      .attr('data-text-class', 'center')
+
+    points.neckCp1 = points.neckRight.shift(90, points.neckFront.dy(points.neckRight) * 0.8)
+    points.neckCp2 = points.neckFront.shift(180, points.neckFront.dy(points.neckRight) * 0.8)
+
+    points.neckCp3 = points.neckLeft.shift(90, points.neckFront.dy(points.neckLeft) * 0.8)
+    points.neckCp4 = points.neckFront.shift(0, points.neckFront.dy(points.neckLeft) * 0.8)
+
+    paths.neckLineFront = new Path()
+    .move(points.neckRight)
+      .curve(points.neckCp1, points.neckCp2, points.neckFront)
+      .curve(points.neckCp4, points.neckCp3, points.neckLeft)
+
+    points.neckBack = points.middle
+      .shiftTowards(points.neckFront, measurements.hpsToBust * options.neckDepthBack)
+      .attr('data-text', 'Back Neckline')
+      .attr('data-text-class', 'center')
+
+    points.neckBackCp1 = points.neckBack.shift(0, points.neckFront.dy(points.neckLeft) * 0.8)
+    points.neckBackCp2 = points.neckBack.shift(180, points.neckFront.dy(points.neckRight) * 0.8)
+    points.neckBackCp3 = points.neckLeft.shift(90, points.neckBack.dy(points.neckLeft) * 0.8)
+    points.neckBackCp4 = points.neckRight.shift(90, points.neckBack.dy(points.neckRight) * 0.8)
+
+    paths.necklineBack = new Path()
+      .move(points.neckRight)
+      .curve(points.neckBackCp4, points.neckBackCp2, points.neckBack)
+      .curve(points.neckBackCp1, points.neckBackCp3, points.neckLeft)
+
+    points.topTopLeft = points.topLeft.shift(90, 20)
+    points.topTopRight = points.topRight.shift(90, 20)
+    paths.extraBoundary = new Path()
+      .move(points.topLeft)
+      .move(points.topRight)
+      .move(points.topTopRight)
+      .move(points.topTopLeft)
+      .move(points.topLeft)
+      .close()
+
+      points.logo = points.middle.shiftTowards(points.middleHem, maxLength / 4)
+      snippets.logo = new Snippet('logo', points.logo)
+
+      points.title = points.logo.shiftTowards(points.middle, maxLength / 12)
+      macro('title', {
+        at: points.title,
+        nr: 1,
+        title: 'body',
+      })
+
+      if (sa) {
+        paths.sa = paths.sideseam.offset(sa).setClass('fabric sa')
+      }
+
+      macro('cutonfold', {
+        from: points.topLeft,
+        to: points.topRight,
+        grainline: false,
+      })
+     
+      points.grainlineFrom = points.leftShoulder
+      points.grainlineTo = new Point(points.grainlineFrom.x, points.bottomLeft.y)
+      macro('grainline', {
+        from: points.grainlineFrom,
+        to: points.grainlineTo,
+      })
+    }
```

Comment:
Here we go, you're closing a non-existing brace here because your indentation is off.

Note that his code does not even build like this. Running `yarn build` in your design can spot these things.
Also note that Deepscan will catch these kind of errors.

Reply:
This is a missed bracket removal from when removing if complete.